crypto/tbs/src/lib.rs

```diff
@@ -265,4 +268,24 @@ mod tests {
         let sig = unblind_signature(bkey, bsig);
         assert!(verify(msg, sig, pk));
     }
+
+    #[test]
+    #[should_panic(expected = "Not enough signature shares")]
+    fn test_insufficient_shares() {
+        let msg = Message::from_bytes(b"Hello World!");
+        let threshold = 5;
+
+        let (_, bmsg) = blind_message(msg);
+
+        let (_, _pks, sks) = dealer_keygen(threshold, 4);
```

Comment:
That's a damn clever way of generating the test case :laughing: this call should also panic (t>n), but I guess dealer keygen isn't meant to run in production so let's leave it at that.